test/OSMTests.js

```diff
@@ -6,12 +6,11 @@ import {
 
 import Adapter from 'enzyme-adapter-react-16';
 
-import { expect } from 'chai';
+// import { expect } from 'chai';
```

Comment:
Can we just delete?

Reply:
I added a test that uses this, so... uncommented rather than deleted :)